main.tf

```diff
@@ -231,6 +231,20 @@ resource "google_compute_address" "static_nginx_ip" {
   }
 }
 
+resource "helm_release" "nginx_controller" {
+  count            = var.nginx_controller.enabled ? 1 : 0
+  name             = "ingress-nginx"
```

Comment:
Nope, let it be `ingress-nginx` following the chart name. 
`nginx-ingress` is actually a depreciated older version which in moved to `ingress-nginx`. So to avoid further confusion better to stick with latest naming standards.

Reply:
I don't know the history, but i guess in helm community `nginx-ingress` has a scary past 😅

Reply:
Fair point, but when it comes to naming conventions I kinda want to go with the chart's name. Even the official kubernetes repo refers to it as ingress-nginx and not the other way around. https://github.com/kubernetes/ingress-nginx
Let @dibosh be the tie breaker here 😬

Reply:
The kubernetes github account names its **_repo URL_** as ingress-nginx simply for indexing purposes. When the kubernetes account in github has so many repositories of so many kinds, it makes sense to have its repos sorted by their **kinds** first. Hence, the word "ingress" is prefixed instead of being suffixed.

But notice how they refer to the controller in the page inside the URL. Everywhere it's being referred to as "NGINX ingress controller" or as "NGINX controller". Because that's what the official **_name_** of the product is.

Since we are trying to **_name_** our components here, we should go with the name of the product rather than how it is indexed in different places.

@dibosh share your views here please

Reply:
I am fine with `nginx-ingress-contoller`

Reply:
`ingress` and `ingress-controller` is two different things. What you are creating with the helm chart will be a type of Ingress Controller that is based on Nginx. So it makes sense to name it `nginx-ingress-controller`.